### PR TITLE
fix: Markdownlint cleanup for RU

### PR DESCRIPTION
**Pull request by nschonni**
Part two of the cleanup

### CODE REVIEW

**Comment by SphinxKnight** [comment]
@mdn/yari-content-ru could you please have a look?
If there is no veto or review in the 15 coming days, I'll merge this one after this time.

File: files/ru/web/css/css_grid_layout/realizing_common_layouts_using_css_grid_layout/index.md [hunk 1]
```diff
@@ -439,7 +439,7 @@ translation_of: Web/CSS/CSS_Grid_Layout/Realizing_common_layouts_using_CSS_Grid_
 
 ![The layout has gaps as there is not space to layout a two track item.](https://mdn.mozillademos.org/files/14751/11-grid-auto-flow-sparse.png)
 
-Я могу привести причину недостатка заполнения с помощью {{cssxref("grid-auto-flow")}}`: dense `в грид ячейке. Будьте осторожны, когда делаете это, поскольку это действительно уводит элементы от их логического исходного порядка. Вы должны делать это только в том случае, если ваши элементы не имеют установленного порядка – и быть в курсе проблем порядка вкладок после источника, а не вашего переупорядоченного отображения.
+Я могу привести причину недостатка заполнения с помощью {{cssxref("grid-auto-flow")}}`: dense` в грид ячейке. Будьте осторожны, когда делаете это, поскольку это действительно уводит элементы от их логического исходного порядка. Вы должны делать это только в том случае, если ваши элементы не имеют установленного порядка – и быть в курсе проблем порядка вкладок после источника, а не вашего переупорядоченного отображения.
```
Comment (sashasushko) [inline files/ru/web/css/css_grid_layout/realizing_common_layouts_using_css_grid_layout/index.md:442]:
```suggestion
Я могу привести причину недостатка заполнения с помощью {{cssxref("grid-auto-flow")}}: `dense` в грид ячейке. Будьте осторожны, когда делаете это, поскольку это действительно уводит элементы от их логического исходного порядка. Вы должны делать это только в том случае, если ваши элементы не имеют установленного порядка – и быть в курсе проблем порядка вкладок после источника, а не вашего переупорядоченного отображения.
```

File: files/ru/web/css/css_positioning/understanding_z_index/stacking_without_z-index/index.md [hunk 2]
```diff
@@ -19,8 +19,9 @@ original_slug: Web/Guide/CSS/Understanding_z_index/Stacking_without_z-index
 
 В следующем примере, абсолютно и относительно спозиционированным блокам определена величина и позиция таким образом, чтобы продемонстрировать правила наложения.
 
-> **Примечание:** **Примечания:\*** Given a homogeneous group of elements without any z-index property, such as the positioned blocks (DIV #1 to #4) in the example, the element's stacking order is their order in the HTML hierarchy, regardless of their position.
+> **Примечание:** **Примечания:**
```
Comment (sashasushko) [inline files/ru/web/css/css_positioning/understanding_z_index/stacking_without_z-index/index.md:22]:
```suggestion
> **Примечания:**
```

File: files/ru/web/css/display-listitem/index.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ translation_of: Web/CSS/display-listitem
 
 Единственное значение `list-item приведёт к тому, что элемент будет вести себя как элемент списка. Это можно использовать вместе с {{CSSxRef("list-style-type")}} и {{CSSxRef("list-style-position")}}.`
 
-> **Примечание:** `Примечание:В браузерах, поддерживающих двухзначный синтаксис, если внутреннее значение не указано, по умолчанию оно будет `flow`. Если внешнее значение не указано, основной блок будет иметь внешний вид отображения (`outer display type`) block.`
+> **Примечание:** Примечание:В браузерах, поддерживающих двухзначный синтаксис, если внутреннее значение не указано, по умолчанию оно будет `flow`. Если внешнее значение не указано, основной блок будет иметь внешний вид отображения (`outer display type`) block.`
```
Comment (sashasushko) [inline files/ru/web/css/display-listitem/index.md:21]:
```suggestion
> **Примечание:** В браузерах, поддерживающих двухзначный синтаксис, если внутреннее значение не указано, по умолчанию оно будет `flow`. Если внешнее значение не указано, основной блок будет иметь внешний вид отображения (`outer display type`) block.`
```

File: files/ru/web/css/max-block-size/index.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ translation_of: Web/CSS/max-block-size
...
+[CSS](/ru/docs/Web/CSS) свойство** `max-block-size`** определяет максимальный горизонтальный или вертикальный размер блока в зависимости от вида ориентации экрана. Это свойство соответствует либо {{cssxref("max-width")}}, либо {{cssxref("max-height")}}, в зависимости от значения определённого в {{cssxref("writing-mode")}}. Если последнее свойство вертикально ориентировано, значит значение свойства `max-block-size` относится к максимальной ширине блока, в противном случае относится к максимальной высоте блока. It relates to {{cssxref("max-inline-size")}}, which defines the other dimension of the element.
```
Comment (sashasushko) [inline files/ru/web/css/max-block-size/index.md:8]:
```suggestion
[CSS](/ru/docs/Web/CSS) свойство **`max-block-size`** определяет максимальный горизонтальный или вертикальный размер блока в зависимости от вида ориентации экрана. Это свойство соответствует либо {{cssxref("max-width")}}, либо {{cssxref("max-height")}}, в зависимости от значения определённого в {{cssxref("writing-mode")}}. Если последнее свойство вертикально ориентировано, значит значение свойства `max-block-size` относится к максимальной ширине блока, в противном случае относится к максимальной высоте блока. It relates to {{cssxref("max-inline-size")}}, which defines the other dimension of the element.
```

File: files/ru/web/css/text-size-adjust/index.md [hunk 5]
```diff
@@ -21,7 +21,7 @@ But when the user zoomed in until the text becomes readable, an horizontal scrol
 
 The `text-size-adjust` property allows Web authors to opt out from this behavior, as Web pages designed to handle small screen widths do not need it.
 
-> **Примечание:** **Предупреждения\*\***:\*\*\* Это свойство является нестандартным. In order to use it, Web authors have to duplicate, or even triplicate it, prefixed for each engine supporting it.
+> **Примечание:** **Предупреждения**: Это свойство является нестандартным. In order to use it, Web authors have to duplicate, or even triplicate it, prefixed for each engine supporting it.
```
Comment (sashasushko) [inline files/ru/web/css/text-size-adjust/index.md:24]:
```suggestion
>  **Предупреждение**: Это свойство является нестандартным. In order to use it, Web authors have to duplicate, or even triplicate it, prefixed for each engine supporting it.
```

**Review by sashasushko — APPROVED** [review]

**Comment by SphinxKnight** [comment]
@sashasushko do you have merge privileges? if so, please don't hesitate to merge when you think a PR is OK.
If not, please let me know :)

**Comment by sashasushko** [comment]
@SphinxKnight yes, I have. I was not sure, that I should merge pull requests about migration.

But ok. All next PRs I will merge, if them are ok for me. Thanks

**Comment by SphinxKnight** [comment]
@sashasushko thanks for the swift feedback, don't hesitate  to ask if you're unsure anyway :)

**Pull request merged**